### DISCUSSION TITLE
Any reason for the dead shortened link in a comment in the head.html file?

### DISCUSSION THREAD

**Discussion by trwbox** [mmistakes/minimal-mistakes#4421] [Q&A] [answered]
Specifically there the link in the comment at [_include/head.html#9](https://github.com/mmistakes/minimal-mistakes/blob/8a67ce8e41ec850f2d7c373aa47739b2abfee6f1/_includes/head.html#L9) that appears to do nothing, besides being a dead link.

After taking looks at wayback machine it appears to have pointed to this [presentation](https://docs.google.com/presentation/d/1rmxwWa9P6_xHqonmh5ONXRS-jPc5XKbnv99Rjkhe04s/), more specifically the [present](https://docs.google.com/presentation/d/1rmxwWa9P6_xHqonmh5ONXRS-jPc5XKbnv99Rjkhe04s/present) option of the slides. Which is a presentation on viewports and media queries. 

I was just curious if there was a reason for it being there while it is a dead link? If there is no reason, and it's purpose was to provide information about the meta tag below it. Would it make sense to either include a direct link, archive.org link, or a new shortened link?

**Comment by iBug** [comment, marked as answer]
It's just as simple as the link went dead some time in 2023.

Fixed in a7af9b0ad2b4dc1a6424fb59843f6cc69447f5e1.
